README.md

```diff
@@ -35,21 +32,23 @@ Current release info
 | Name | Downloads | Version | Platforms |
 | --- | --- | --- | --- |
 | [![Conda Recipe](https://img.shields.io/badge/recipe-dask-green.svg)](https://anaconda.org/conda-forge/dask) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/dask.svg)](https://anaconda.org/conda-forge/dask) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/dask.svg)](https://anaconda.org/conda-forge/dask) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/dask.svg)](https://anaconda.org/conda-forge/dask) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-dask--core-green.svg)](https://anaconda.org/conda-forge/dask-core) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/dask-core.svg)](https://anaconda.org/conda-forge/dask-core) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/dask-core.svg)](https://anaconda.org/conda-forge/dask-core) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/dask-core.svg)](https://anaconda.org/conda-forge/dask-core) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-distributed-green.svg)](https://anaconda.org/conda-forge/distributed) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/distributed.svg)](https://anaconda.org/conda-forge/distributed) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/distributed.svg)](https://anaconda.org/conda-forge/distributed) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/distributed.svg)](https://anaconda.org/conda-forge/distributed) |
 
-Installing dask
-===============
+Installing dask-split
+=====================
 
-Installing `dask` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+Installing `dask-split` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
 
 ```
 conda config --add channels conda-forge
 conda config --set channel_priority strict
 ```
 
-Once the `conda-forge` channel has been enabled, `dask` can be installed with:
+Once the `conda-forge` channel has been enabled, `dask, dask-core, distributed` can be installed with:
 
 ```
-conda install dask
+conda install dask dask-core distributed
```

Comment:
I think these should all just say `dask` since `dask` installs them all.

Reply:
This is also autogenerated. So not much we can do here.

Reply:
yeah ok I was kind of worried that was the case. I guess it's not too bad becuase it would be a pretty strange user who encountered this as readme as their install instructions. It a little bit seems like this is not the place for install instructions at all.

Reply:
Well it seems reasonable to provide some kind of install instructions on the feedstock given this is where the recipe is maintained and packages published.

While this particular piece is duplicative in our case, it is not always so (there is not always a kitchen sink package to install) and it should still work (despite being a bit verbose).

The portion above with installing `dask-split` is a bit worse IMHO as that is actually providing inaccurate information that seems unlike to ever be correct.

That all being said, if you would like to [raise a conda-smithy issue]( https://github.com/conda-forge/conda-smithy/issues/new ) on how to improve this with some suggestions, would be supportive of that. Whenever conda-smithy has the intended improvement, it should be a simple re-render to replace what is currently here.

Reply:
Yeah I tried to use `package_name` at the time, but it didn't seem to solve the issue.